modules/biobank/php/containerdao.class.inc

```diff
@@ -0,0 +1,534 @@
+<?php declare(strict_types=1);
+/**
+ * Container Data Access Object (DAO) Class
+ * This class contains all database handling that is needed to
+ * permanently store and retrieve Container Object instances
+ *
+ * PHP Version 7.2
+ *
+ * @category   Data
+ * @package    Loris
+ * @subpackage Biobank
+ * @author     Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+namespace LORIS\biobank;
+
+/**
+ * Container Data Access Object (DAO) Class
+ * This class contains all database handling that is needed to
+ * permanently store and retrieve Container Object instances
+ *
+ * PHP Version 7.2
+ *
+ * @category   Data
+ * @package    Loris
+ * @subpackage Biobank
+ * @author     Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+  @link       https://www.github.com/aces/Loris/
```

Comment:
GitHub's diff is highlighting this as an error. I think it wants a '@'

Reply:
fixed